src/config.hpp

```diff
@@ -92,7 +92,7 @@ class Config {
          * 
          * @return std::string 
          */
-    const std::string& configPath() {
+    const std::string& configPath() const {
```

Comment:
You add `const` but in fact this line can modify `this->result`:
```
return result->operator[]("config_path").as<std::string>();
```

In case `config_path` do not exist, it will be added

Reply:
This is not std::map but cxxopts::ParseResult
https://github.com/jarro2783/cxxopts/blob/6bb1e82b6132bd290313851d352b6d02ac81cb44/include/cxxopts.hpp#L1573